src/main/app/claims/models/claim.ts

```diff
@@ -184,6 +185,13 @@ export class Claim {
       return ClaimStatus.MORE_TIME_REQUESTED
     } else if (!this.response) {
       return ClaimStatus.NO_RESPONSE
+    }
+    if (this.hasCCJ()) {
```

Comment:
you shouldn't need this here. Put the check for hasCCJ into the isCCJPaidWithinMonth and maybe rename that method to hasCCJBeenPainInOneMonth ?